assets/src/components/v2/departures/header.tsx

```diff
@@ -23,13 +24,31 @@ const DirectionArrow = ({ arrow }: { arrow: CardinalDirection }) => (
   />
 );
 
-const Header = ({ title, arrow }: Header) => {
+const Header = ({ title, arrow, subtitle }: Header) => {
+  const formatted_subtitle = format_subtitle(subtitle);
+
   return (
-    <header className="departures-header">
-      {(title || arrow) && <span>{title}</span>}
-      {arrow && <DirectionArrow arrow={arrow} />}
-    </header>
+    <div>
+      <header className="departures-header">
+        {(title || arrow) && <span>{title}</span>}
+        {arrow && <DirectionArrow arrow={arrow} />}
+      </header>
+      {formatted_subtitle && (
+        <div className="departures-header__subtitle">{formatted_subtitle}</div>
```

Comment:
We're sort of doing this "null check" twice, once here and once in `formatSubtitle` itself. Rather than passing a null through and requiring `formatSubtitle` to handle it, I think we could simplify this if we did:

```tsx
{subtitle && <div ...>{formatSubtitle(subtitle)}</div>}
```

Reply:
nice simplification!